java/core/src/test/java/co/worklytics/psoxy/rules/github/GitHubTests.java

```diff
@@ -533,7 +533,7 @@ void pulls() {
 
         String sanitized = this.sanitize(endpoint, jsonString);
 
-        assertNotSanitized(sanitized, "octocatorg");
+        assertPseudonymized(sanitized, "octocatorg");
```

Comment:
That's the name of the `owner`  in the example (because is an organization user)